Tasks/Npm/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -10,6 +10,7 @@
   "loc.input.help.arguments": "Additional arguments passed to npm.",
   "loc.input.label.cwd": "Working Directory",
   "loc.input.help.cwd": "Current working directory where npm command is run.  Defaults to the root of the repo.",
+  "loc.messages.InvalidCommand": "Illegal characters in command '%s'. Command cannot contain a space.",
```

Comment:
Only one command should be used for npm.  Use arguments input for additional arguments.